src/components/Calculator.js

```diff
@@ -1,7 +1,6 @@
 import { useState } from 'react';
 import calculate from '../logic/calculate';
 import CalcButtons from './CalcButtons';
-import FetchQuote from './FetchQuote';
 
 export default function Calculator() {
   const [calculatorDataObj, setCalculatorDataObj] = useState({
```

Comment:
- [ ] Please kindly design this page according to the wireframe. Wireframes help ensure that designers, developers, and stakeholders are aligned on the product's structure and functionality. ℹ️ 

<img width="725" alt="Untitled" src="https://github.com/VaheStepanyan100/math-magicians/assets/103386183/977c44cc-4665-4781-bdca-317b08657a83">